test/file-header_test.go

```diff
@@ -27,4 +27,20 @@ func TestLintFileHeader(t *testing.T) {
 	testRule(t, "lint-file-header5", &rule.FileHeaderRule{}, &lint.RuleConfig{
 		Arguments: []interface{}{"^\\sfoo.*bar$"},
 	})
+
+	testRule(t, "lint-file-header6", &rule.FileHeaderRule{}, &lint.RuleConfig{
+		Arguments: []interface{}{"foobar"},
+	})
+}
+
+func BenchmarkLintFileHeader(b *testing.B) {
+	benchRule(b, "lint-file-header1", &rule.FileHeaderRule{}, &lint.RuleConfig{
+		Arguments: []interface{}{"foobar"},
+	})
+	//var t *testing.T
```

Comment:
please avoid commented-out code in commits

Reply:
Done